categories/99-problems/P40-rhebus.pl

```diff
@@ -35,6 +35,6 @@ (Even $n where * > 2 )
     return; # fail
 }
 
-say goldbach $_ for 28, 36, 52, 110, 62710560;
+say ~ goldbach $_ for 28, 36, 52, 110;
```

Comment:
On Monday, May 2, 2016, Paul Cochrane <notifications@github.com> wrote:

> In categories/99-problems/P40-rhebus.pl
> <https://github.com/perl6/perl6-examples/pull/43#discussion_r61723165>:
>
> > @@ -35,6 +35,6 @@ (Even $n where * > 2 )
> >      return; # fail
> >  }
> >
> > -say goldbach $_ for 28, 36, 52, 110, 62710560;
> > +say ~ goldbach $_ for 28, 36, 52, 110;
>
> Why was the last input value removed here? Could it be left in?
>

It was removed because I saw no corresponding expected output. I agree that
it should be added back.

-Tom

>